src/securityPolicy.h

```diff
@@ -84,7 +88,8 @@ security_policy_t policyForSignTxStakePoolRegistrationRewardAccount(
 );
 security_policy_t policyForSignTxStakePoolRegistrationOwner(
         const sign_tx_signingmode_t txSigningMode,
-        const pool_owner_t* owner
+        const pool_owner_t* owner,
+        uint16_t numOwnersGivenByPath
```

Comment:
I think we are using `size_t` in other policies for such numbers.

Reply:
I don't think so, there are plenty of examples where we use the original data type of the data.

Reply:
You are right, I was not checking carefully. There are two places where we use size_t:
```
security_policy_t policyForSignTxStakePoolRegistrationInit(
        sign_tx_signingmode_t txSigningMode,
        size_t numOwners
)
```
and in
```
security_policy_t policyForGetPublicKeysInit(size_t numPaths)
```
And some where we use `uint32_t`:
```
security_policy_t policyForSignTxStakePoolRegistrationConfirm(
        uint32_t numOwners, uint32_t numRelays
);
```

Considering that the numbers are serialized as uint32_t in APDUs and we only use uint16 to save memory in ctx, I am in favor of using uint32_t in all the security policies. Would you change it, please? Maybe in a separate PR?